src/boardHelper.ts

```diff
@@ -100,6 +104,21 @@ export const createNewBoard = (): Board => {
 
 export type Move = { type: SquareType; coordinates: Coordinates; userId: string };
 
+export const getPlayerCoordinates = (playerId: string, board: Board): Coordinates | null => {
+  const keys = Object.keys(board);
+  for (let indexY = 0; indexY < keys.length; indexY++) {
```

Comment:
don't use for cycles like this when unnecessary. Instead, use `for (const rowKey of Object.keys(board))`. That way you won't even need the `keys` variable.